trl/trainer/grpo_trainer.py

```diff
@@ -1086,12 +1086,30 @@ def _compute_loss(self, model, inputs):
             mean_kl = (per_token_kl * completion_mask).sum() / completion_mask.sum()
             self._metrics[mode]["kl"].append(self.accelerator.gather_for_metrics(mean_kl).nanmean().item())
 
-        # Compute the clip ratio
-        is_clipped = ((coef_1 < 1 - self.epsilon_low) & (advantages.unsqueeze(1) < 0)) | (
-            (coef_1 > 1 + self.epsilon_high) & (advantages.unsqueeze(1) > 0)
+        # Compute the clipped probability ratios
```

Comment:
@kashif do I also need to update the Liger loss to account for these new metrics?